src/yaml.cpp

```diff
@@ -219,6 +223,8 @@ DR_PARAM_DEFINE_YAML_DECODE(float      , node) { return convert_floating_point<f
 DR_PARAM_DEFINE_YAML_DECODE(double     , node) { return convert_floating_point<double>     (node); }
 DR_PARAM_DEFINE_YAML_DECODE(long double, node) { return convert_floating_point<long double>(node); }
 
+DR_PARAM_DEFINE_YAML_DECODE(YAML::Node, node) { return convert_yaml_node(node); }
```

Comment:
Actually, why the convert_yaml_node here? Why not just return node?

Reply:
Makes sense